test/test_proxy_tensor.py

```diff
@@ -1868,6 +1868,12 @@ def f(t):
 
     # AssertionError: Tensor-likes are not close!
     skip('empty_strided', '', device_type='cpu'),
+
+    # RuntimeError: Cannot call numel() on tensor with symbolic sizes/strides
+    skip('index_reduce', 'prod'),
```

Comment:
These should be `xfail` rather than `skip`.

I guess these failures are expected if there's no meta impl, but do you know why there wasn't an xfail for the non-variant opinfo?

Reply:
> These should be `xfail` rather than `skip`.

Fixed.

> I guess these failures are expected if there's no meta impl, but do you know why there wasn't an xfail for the non-variant opinfo?

yes:
(i) the non-variant opinfo emitted samples for all reduction types
(ii) `_test_make_fx_helper` produces only 5 samples when `out=True` which is the case for `test_make_fx_symbolic_exhaustive_out`.

The first 5 samples contain scalar inputs but the failure requires a non-scalar input. When enabling all other samples by disabling the if-block in https://github.com/pytorch/pytorch/blob/9dfeec9cdc246a6a003dff4b6ba0a5ceb60613f1/test/test_proxy_tensor.py#L2007, the failure can be reproduced on the main branch as well.

That said, the xfail instances is now moved to `out_symbolic_tensor_failures` list.